code/modules/research/designs/medical_designs.dm

```diff
@@ -152,6 +152,28 @@
 	build_path = /obj/item/organ/cyberimp/arm/toolset
 	category = list("Misc", "Medical Designs")
 
+/datum/design/cyberimp_surgical
+	name = "Surgical Toolkit implant"
+	desc = "A full kit of surgical tools that can be implanted into a human's arm."
+	id = "ci-surgical"
+	req_tech = list("materials" = 4, "engineering" = 4, "biotech" = 5, "powerstorage" = 4, "programming" = 3)
+	build_type = PROTOLATHE | MECHFAB
+	materials = list (MAT_METAL = 4500, MAT_GLASS = 1500, MAT_GOLD = 2000, MAT_SILVER = 750)
+	construction_time = 200
+	build_path = /obj/item/organ/cyberimp/arm/surgery
+	category = list("Misc", "Medical Designs")
+
+/datum/design/cyberimp_flash
+	name = "Photon Projector implant"
+	desc = "A fancy name for an implantable self-recharging flashbulb device that doubles has a high intensity flashlight."
+	id = "ci-flash"
+	req_tech = list("materials" = 5, "engineering" = 4, "biotech" = 5, "powerstorage" = 4, "programming" = 4, "plasmatech" = 3, "magnets" = 4, "combat" = 4)
```

Comment:
Looking over your previous PR, I don't think you added origin tech to these implants. Please do.